app/controllers/mentorships_controller.rb

```diff
@@ -7,13 +7,13 @@ def auth_user
 
   def new
     @user = current_user
-    @mentorship = Mentorship.new()
+    @mentorship = Mentorship.new
     @industries = ['Technology', 'Business', 'Startup']
   end
 
   def create
     @user = current_user
-    @mentorship = Mentorship.new(mentee_id: current_user.id, question: params[:mentorship][:question])
+    @mentorship = Mentorship.new mentee_id: current_user.id, question: params[:mentorship][:question]
```

Comment:
let's keep parenthesis here, to stay consistent with [style guide](https://github.com/bbatsov/ruby-style-guide#method-invocation-parens)